wallet/src/bin/web_server.rs

```diff
@@ -327,17 +328,27 @@ mod tests {
         println!("mnemonic: {}", mnemonic);
         let password = "my-password";
 
-        // Should fail if no wallet exists.
-        server
-            .requires_wallet::<()>(&format!("openwallet/{}/path/{}", password, server.path()))
-            .await;
+        // clean up from any previous test run on the current machine
+        let mut storage_path = get_home_path().unwrap();
+        storage_path.push(".espresso/cape/last_wallet_path");
+        remove_file(storage_path).unwrap();
```

Comment:
Actually, can we have an env var to override the default `.espresso` path, and then use a tempdir for this test? That might also be pretty useful for Plaza as they're developing/testing

Reply:
Updated the code based on this, using /tmp for the test